features/step_definitions/metadata.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const request = require('../support/request');
+const TIMEOUT = 240 * 1000;
+
+module.exports = function server() {
+    // eslint-disable-next-line new-cap
+    this.Before({
+        tags: ['@metadata']
+    }, () => {
+        this.repoOrg = '';
+        this.repoName = 'functional-metadata';
+        this.pipelineId = null;
+        this.eventId = null;
+        this.meta = null;
+        this.jwt = null;
+    });
+
+    this.Given(/^a metadata starts with an empty object$/);
+
+    this.Then(/^the "BAR" job is started$/, { timeout: TIMEOUT }, () =>
```

Comment:
`the "BAR" job is started` case and `the "BAZ" job is started` case are looks similar.
Maybe these can be changed like:
https://github.com/screwdriver-cd/screwdriver/blob/master/features/step_definitions/templates.js#L9